File: javascript/scrape.js
const axios = require('axios');
const cheerio = require('cheerio');

const url = 'https://www.example.com/';

axios.get(url)
    .then(response => {
        const $ = cheerio.load(response.data);
        const pTagText = $('p').text();
        console.log(`Text inside <p> tags: ${pTagText}`);
    })
    .catch(error => console.error('Error:', error));

File: kotlin/scrape.kt
import org.jsoup.Jsoup

fun main() {
    val url = "https://www.example.com/"
    try {
        val document = Jsoup.connect(url).get()
        val pTags = document.select("p")
        val pTagText = pTags.text()
        println("Text inside <p> tags: $pTagText")
    } catch (e: Exception) {
        e.printStackTrace()
    }
}
